app/views/steps/client/nino_exit/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<div class="flash-card">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <header>
+        <h1 class="govuk-heading-l"><%= t('.heading') %></h1>
+      </header>
+
+      <p>You cannot use this service if your client:</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>has a partner</li>
+        <li>does not have a National Insurance number </li>
+        <li>does not receive a passporting benefit based on DWP records, including Universal Credits</li>
+        <li>is employed, including self-employed, business partner, company director or shareholder</li>
+      </ul>
+
+      <div>
+        <%= link_to t('.eforms_apply'), t('.eforms_url'), class: 'govuk-button' %>
```

Comment:
There is a `#link_button` helper to makes links look like buttons that might be useful here.